baud: 115200
devices: [ "/dev/robot/ttyTrex0" ]
rate: 50

SerialTimeout: 0        #0: no timeout, otherwise: SerialTimeout*0.1s, max=13s
Motor1PWMPrescaler: 3
Motor2PWMPrescaler: 3
